projects/grok/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/GrokImageCompression/grok"
+primary_contact: "boxerab@gmail.com"
+fuzzing_engines:
+  - libfuzzer
+  - afl
+  - dataflow
```

Comment:
I'd recommend removing `dataflow` engine and sanitizer for now, as they require instrumenting all the dependencies and that's why the build is failing

Reply:
Gotcha. Thanks. Now removed.